ethereum-relayer/Makefile

```diff
@@ -0,0 +1,51 @@
+SOURCE_FILES:=$(shell find contracts -name "*.sol")
+
+.PHONY: dependencies test clean all
+
+all: build
+
+node_modules: package-lock.json
+	touch -m node_modules
+	npm ci
+
+# Note: Forge really wants to manage dependencies via submodules, but that
+# workflow is a little awkward. There's currently no support for a more
+# traditional package manager workflow (with a package manifest file and
+# installation into a subdirectory that can be gitignored).
+# Instead, we just specify the dependencies here. make will then take care of
+# installing them if they are not yet present.
+# When adding a new dependency, make sure to specify the exact commit hash, and
+# the --no-git and --no-commit flags (see lib/forge-std below)
+.PHONY: forge_dependencies
+forge_dependencies: lib/forge-std lib/openzeppelin-contracts lib/wormhole-solidity-sdk
+
+lib/forge-std:
+	forge install foundry-rs/forge-std@v1.5.5 --no-git --no-commit
+
+lib/openzeppelin-contracts:
+	forge install openzeppelin/openzeppelin-contracts@0457042d93d9dfd760dbaa06a4d2f1216fdbe297 --no-git --no-commit
+
+lib/wormhole-solidity-sdk:
+	forge install wormhole-foundation/wormhole-solidity-sdk --no-git --no-commit
```

Comment:
Should there be a tag for this?

Reply:
Probably the latest commit as of right now?